tools/scripts/zod-to-schema.ts

```diff
@@ -0,0 +1,69 @@
+import { writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+import yargs from 'yargs';
+import { z } from 'zod';
+import { zodToJsonSchema } from 'zod-to-json-schema';
```

Comment:
Should be installed with `npm i -D zod-to-json-schema`.

Reply:
I will delete this script

Reply:
but for now reverted the changes